src/blueapi/cli/cli.py

```diff
@@ -101,6 +118,7 @@ def start_application(obj: dict):
 def controller(ctx: click.Context, output: str) -> None:
     """Client utility for controlling and introspecting the worker"""
 
+    setup_tracing("BlueAPICLI", OTLP_EXPORT_ENABLED)
```

Comment:
Will these all appear as the same "BlueAPICLI" in Jaeger or will there be a new one for every time someone runs a `controller` command. If the latter, I'm a bit concerned about noise.

Reply:
So there will only be one BlueAPICLI, but each new run will generate a new Trace, which is desirable so that you can easily identify the run that caused any problems.